### DISCUSSION TITLE
CPU FREQUENCY FOR POWERT SETING SILENT/BALANCED/TURBO

### DISCUSSION THREAD

**Discussion by Viper3000** [seerge/g-helper#1973] [Ideas]
![Max Frekvece for CPU](https://github.com/seerge/g-helper/assets/144729297/1565dc3f-1e57-48d7-8704-8db606639092)

I think it would be interesting to add the option to set the maximum frequency for the CPU as it is in Windows

**Comment by seerge** [comment]
@Viper3000 hello, i was already answering this question in this topic https://github.com/seerge/g-helper/discussions/1907
